knowledge-base/kb-changelocalization-at-runtime.md

```diff
@@ -0,0 +1,42 @@
+---
+title: PDFViewer Localization of The FindDialog During Runtime
+description: "How to change the localization at runtime."
+type: how-to
+page_title: How to change the localization of the FindDialog during runtime for WPF.
+slug: kb-changelocalization-at-runtime
+position: 0
+tags: PDFViewer, FindDialog, localization, runtime
+ticketid: 1619188
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product Version</td>
+            <td>2023.2.718</td>
+        </tr>
+        <tr>
+            <td>Product</td>
+            <td>RadPDFViewer for WPF</td>
```

Comment:
The correct naming of the control is "RadPdfViewer". Not "RadPDFViewer". My suggestion would be to update every occurrence of this word in the file.